ui/components/src/AxisLabel.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Box, Typography } from '@mui/material';
+
+interface AxisLabelProps {
+  name: string;
+  height: number;
+}
+
+export function AxisLabel({ name, height }: AxisLabelProps) {
```

Comment:
Not sure, but if this is always rotated to the left, should we call it `YAxisLabel` (it would only apply to y-axis?)

Reply:
Yeah good call, I just changed the component name!